docs/install/win/activating-a-validator.md

```diff
@@ -3,43 +3,41 @@ id: activating-a-validator
 title: Activating a Validator
 sidebar_label: Activate a validator
 ---
-This section outlines the step-by-step process found on prylabs.net to submit a deposit and initialise a validator for participation in the testnet.
+This section outlines the step-by-step process for Windows found on prylabs.net to submit a deposit and initialise a validator for participation in the testnet.
 
 
-## Downloading Prysm
+## Step 1: Get Prysm
 
-To begin, follow the instructions found in the [installation section](./#installing-prysm) of the [Getting Started](./) guide to fetch and install Prysm with either Docker or Bazel.
+To begin, follow the instructions to fetch and install Prysm with either the [Prysm Installation Script](../windows), [Docker](./docker) or [Bazel](./bazel).
 
-## Receiving Göerli ETH
+## Step 2: Get Göerli ETH - Test ether
 
-On step 2, you will be asked to link a wallet address to your validator with either the [Metamask](https://metamask.io/) browser extension \(recommended\) or [Portis](https://portis.io). Select your preferred platform and click through the steps presented.
+You will be asked to link a wallet address to your validator with either the [Metamask](https://metamask.io/) browser extension \(recommended\) or [Portis](https://portis.io). Select your preferred platform and click through the steps presented.
 
 ![](https://blobscdn.gitbook.com/v0/b/gitbook-28427.appspot.com/o/assets%2F-LRNnKRqTm4z1mzdDqDF%2F-LuJpxGKxOpat8TfDxPP%2F-Lua4msnLMulYW-XYrN_%2F2.png?alt=media&token=8268d6b5-97da-414a-9110-141a7aaeb3de)
 
 The wallet is scanned for the required amount of Göerli ETH after being linked. If the wallet does not have sufficient funds, you will be given the option to receive the required GöETH from our faucet.
 
-## Generating a validator keypair
+## Step 3: Generating a validator keypair
 
-Step 3 requires running a command to generate a public / private keypair for your validator, as well as deposit data to submit on the prylabs.net page. Depending on your platform, issue the appropriate command from the examples below.
+You need to run a command to generate a public / private keypair for your validator.  Depending on your platform, issue the appropriate command from the examples below.
```

Comment:
We generally avoid using 'you', 'your' etc. throughout the main documentation, with the exception of contribution guides and more informal sections

Reply:
Ok, will adjust the style accordingly